service/storage_service.cc

```diff
@@ -1469,16 +1469,9 @@ void storage_service::update_peer_info(gms::inet_address endpoint) {
 std::unordered_set<locator::token> storage_service::get_tokens_for(inet_address endpoint) {
     auto tokens_string = _gossiper.get_application_state_value(endpoint, application_state::TOKENS);
     slogger.trace("endpoint={}, tokens_string={}", endpoint, tokens_string);
-    if (tokens_string.size() == 0) {
-        return {}; // boost::split produces one element for emty string
-    }
-    std::vector<sstring> tokens;
-    std::unordered_set<token> ret;
-    boost::split(tokens, tokens_string, boost::is_any_of(";"));
-    for (auto str : tokens) {
-        auto t = dht::token::from_sstring(str);
-        slogger.trace("endpoint={}, token_str={} token={}", endpoint, str, t);
```

Comment:
Was token_str not interesting?

Reply:
The `token={}` part should always print the same thing as `token_str={}` part.
If not, then there's still `tokens_string`, which contains everything that `token_str={}` would print, but joined using `;`.
In any case, the `token_str={}` part is redundant.

Reply:
But that's not the reason I removed `token_str={}`. The reason is that I don't have access to `slogger` anymore.